docs/l3/api_flow.md

```diff
@@ -525,4 +525,97 @@ Note: A demand can be commented on multiple times and it does not change the dem
   }
 ```
 
-Note: At any time after a `match2` is proposed, and before it reaches `acceptedFinal` state, any of its members can reject the `match2` using `POST /v1/match2/{id}/rejection`. Once a `match2` is rejected, it is permanently closed.
\ No newline at end of file
+Note: At any time after a `match2` is proposed, and before it reaches `acceptedFinal` state, any of its members can reject the `match2` using `POST /v1/match2/{id}/rejection`. Once a `match2` is rejected, it is permanently closed.
+
+## Cancelling match2
+
+1. Viewing all `match2` cancellation transactions can be performed by `memberA` or `memberB` by making **GET** request to `/v1/match2/{match2Id}/cancellation` endpoint. Takes a `match2id` as a parameter property. Match2 must be in `finalAcceptance` state.
```

Comment:
Surely this will only let you view a single cancellation transaction rather than ALL match2 cancellation transactions?

Reply:
there can be scenario where it could be multiple for a same match2 @mattdean-digicatapult might be able to provide more context. I think due to race condition but memory is a bit rusty

Reply:
I'm going to suggest adding a step for actually cancelling the match2 which is missing from the doc, in addition to just viewing a cancelled match.